61/ShubhamAshish/requirements.txt

```diff
@@ -0,0 +1,3 @@
+streamlit
+pyperclip

```

Comment:
I love this package, using it for pybites-carbon to make a carbon image from code on clipboard. Next time you could pin the requirements.